crates/re_query/src/latest_at/results.rs

```diff
@@ -89,6 +89,24 @@ impl LatestAtResults {
             &EMPTY
         }
     }
+
+    /// Utility for retrieving a single instance of a component.
+    pub fn get_instance<T: re_types_core::Component>(
+        &self,
+        resolver: &PromiseResolver,
+        index: usize,
+    ) -> Option<T> {
+        self.get(T::name())
+            .and_then(|r| r.try_instance::<T>(resolver, index))
+    }
+
+    /// Utility for retrieving a specific component.
+    pub fn get_dense<T: re_types_core::Component>(
```

Comment:
there's a name clash with an extension that lives in re_space_view. Kinda asking for trouble